fl4health/utils/functions.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Tuple
+
+import torch
+
+
+class BernoulliSample(torch.autograd.Function):
+    @staticmethod
+    def forward(bernoulli_probs: torch.Tensor) -> torch.Tensor:  # type: ignore
```

Comment:
I ignored checking the type annotation because the parent class `torch.autograd.Function` is quite loose with its type annotations, while here we are certain that the input should be a tensor.